aptos-move/vm-genesis/src/lib.rs

```diff
@@ -604,19 +641,15 @@ pub fn generate_genesis_change_set_for_testing(genesis_options: GenesisOptions)
         }
     };
 
-    generate_test_genesis(framework, Some(1)).0
-}
-
-/// Generate a genesis `ChangeSet` for mainnet
-pub fn generate_genesis_change_set_for_mainnet(genesis_options: GenesisOptions) -> ChangeSet {
```

Comment:
Outdated mainnet genesis code that has been replaced with a different logic now.